apps/passport-server/test/frogcrypto.spec.ts

```diff
@@ -124,6 +126,31 @@ describe("frogcrypto functionality", function () {
     await testGetFrogFail(feed, DATE_EPOCH_1H, "Frog Not Found");
   });
 
+  it("should update user state after getting frog", async () => {
+    const feed = FROGCRYPTO_FEEDS[0];
+    expect(feed.active).to.be.true;
+    expect(feed.private).to.be.false;
+
+    let userState = await getUserState();
+    expect(userState.success).to.be.true;
+    expect(userState.value?.frogs?.count).to.be.greaterThan(0);
```

Comment:
suggest checking, here and elsewhere, the exact number, since you know the number. eg. `.to.eq(testFrogs.length)`